build/Dockerfile.multi-archs

```diff
@@ -0,0 +1,60 @@
+# Building base image
+FROM --platform=$BUILDPLATFORM golang:1.20.3 AS builder
+ARG version
+ENV KUBETURBO_VERSION $version
+WORKDIR /workspace
+ADD . ./
+RUN make build
+
+
+FROM registry.access.redhat.com/ubi8
+MAINTAINER Turbonomic <turbodeploy@turbonomic.com>
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
+
+RUN echo "Running on $BUILDPLATFORM, and building for $TARGETPLATFORM"
+
+### Atomic/OpenShift Labels - https://github.com/projectatomic/ContainerApplicationGenericLabels
+LABEL name="Hybrid Cloud Container" \
```

Comment:
Does this section need to change?   It seems specific to `suspend` team

Reply:
'Hybrid Cloud Container' === suspend

Reply:
That section is copied from our current [Dockerfile](https://github.com/turbonomic/kubeturbo/blob/50fa8ecba43289b66bae2163756782bd87b34c48/build/Dockerfile#L8), it was written by Enlin 5 years ago based on git history.  I could change it if you want it.